lib/media/media_source_engine.js

```diff
@@ -223,13 +223,14 @@ shaka.media.MediaSourceEngine.prototype.destroy = function() {
  *   'video': 'video/webm; codecs="vp9"', 'text': 'text/vtt' }.
  *   All types given must be supported.
  * @param {boolean} useRelativeCueTimestamps
+ * @param {Object} overrideCueRegion
  *
  * @throws InvalidAccessError if blank MIME types are given
  * @throws NotSupportedError if unsupported MIME types are given
  * @throws QuotaExceededError if the browser can't support that many buffers
  */
 shaka.media.MediaSourceEngine.prototype.init =
-    function(typeConfig, useRelativeCueTimestamps) {
+    function(typeConfig, useRelativeCueTimestamps, overrideCueRegion) {
```

Comment:
If overrideCueRegion is supplied at MediaSourceEngine.init, it cannot be changed mid-playback.  Therefore, there's no reason to supply it to StreamingEngine.configure, and it should be documented that these cue settings only apply to future calls to player.load().